mycli/main.py

```diff
@@ -1001,6 +1004,8 @@ def get_last_query(self):
               help='Use DSN configured into the [alias_dsn] section of myclirc file.')
 @click.option('--list-dsn', 'list_dsn', is_flag=True,
         help='list of DSN configured into the [alias_dsn] section of myclirc file.')
+@click.option('--list-ssh-config', 'list_ssh_config', is_flag=True,
+              help='list of ssh configurations in the ssh config.')
```

Comment:
Can you edit the help message to say "list ssh configurations in the ssh config (requires paramiko)." 

Include the paramiko requirement to the other help messages as well.